### DISCUSSION TITLE
조건형 상품 주문 상세 조회 문입니다.

### DISCUSSION THREAD

**Discussion by limchangtaek** [commerce-api-naver/commerce-solutions#42] [묻고 답하기]
안녕하세요. 굿스플로 임창택입니다.

조건형 상품 주문 상세 조회에서 어느 경우에 remainTotalPaymentAmount가 없나요?
- traceId: cr3-165333-6nd98k^1729065284813^114028740

감사합니다.

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

대상 상품주문번호의 API 호출 시점 결제 금액 (할인 적용 후 금액)을 여쭤보신 것이라면 정확한 필드 이름은 `remainPaymentAmount`이 맞습니다.
(기존 `totalPaymentAmount` 필드의 역할을 각 (1)`initialPaymentAmount` 필드, (2)`remainPaymentAmount` 필드가 나누어 승계)

언급해주신 호출건에서도 `remainPaymentAmount` 필드와 그 값이 제공된 것으로 보입니다.
